subsys/bluetooth/controller/ll_sw/ull_internal.h

```diff
@@ -14,6 +14,7 @@ static inline void ull_hdr_init(struct ull_hdr *hdr)
 	hdr->disabled_cb = hdr->disabled_param = NULL;
 }
 
+int is_bt_hci_cmd_disallowed(void *conn);
```

Comment:
Rename the function ull_is_hci_cmd_disallowed, as this is only used within the ull and also matches the existing naming of other ull functions.

Reply:
Done